tests/tests/test_env_4_user_not_in_configured_teams.py

```diff
@@ -2,37 +2,79 @@
     def configure_env(self):
         assert load_env(__file__)
 
-    @pytest.fixture(autouse=True, scope='class')
+    @pytest.fixture(autouse=True, scope="class")
     def test_config(self):
         return TestConfig.getDefaultTestConfig()
 
-    def test_kssh_no_config_files(self, test_config):
-        # Test that it can't find any config files
-        with outputs_audit_log(test_config, filename="/shared/ca.log", expected_number=0):
-            for s in ['user@sshd-staging', 'root@sshd-staging', 'user@sshd-prod', 'root@sshd-prod']:
+    def test_kssh_no_configs(self, test_config):
+        # Test that it can't find any configs
+        with outputs_audit_log(
+            test_config, filename="/shared/ca.log", expected_number=0
+        ):
+            for s in [
+                "user@sshd-staging",
+                "root@sshd-staging",
+                "user@sshd-prod",
+                "root@sshd-prod",
+            ]:
                 try:
-                    run_command_with_agent(f"""bin/kssh -q -o StrictHostKeyChecking=no {s} "sha1sum /etc/unique" """)
+                    run_command_with_agent(
+                        f"bin/kssh -q -o StrictHostKeyChecking=no {s} "
+                        f'"sha1sum /etc/unique" '
+                    )
                     assert False
                 except subprocess.CalledProcessError as e:
-                    assert b"Did not find any config files in KBFS" in e.output
+                    assert b"Did not find any configs" in e.output
 
     def test_kssh_spoofed_config(self, test_config):
-        # Test that even when kssh is forced to run by a spoofed config, the CA bot ignores messages that are in the
-        # wrong channel
-        with outputs_audit_log(test_config, filename="/shared/ca.log", expected_number=0):
-            client_config = json.dumps({'teamname': f"{test_config.subteam}.ssh", "channelname": "", "botname": test_config.bot_username})
-            run_command(f"echo '{client_config}' | keybase fs write /keybase/team/{test_config.subteam}.ssh/kssh-client.config")
-            for s in ['user@sshd-staging', 'root@sshd-staging', 'user@sshd-prod', 'root@sshd-prod']:
+        # Test that even when kssh is forced to run by a spoofed config, the CA
+        # bot ignores messages that are in the wrong channel
+        with outputs_audit_log(
+            test_config, filename="/shared/ca.log", expected_number=0
+        ):
+            client_config = json.dumps(
+                {
+                    "teamname": f"{test_config.subteam}.ssh",
+                    "channelname": "",
+                    "botname": test_config.bot_username,
+                }
+            )
+            run_command(
+                f'echo \'{{"method":"put", "params": {{'
+                f'"options": {{"team": "{test_config.subteam}.ssh", '
+                f'"namespace": "__sshca", "entryKey": "kssh_config",'
+                f'"entryValue": {json.dumps(client_config)} }}}}}}\' | '
```

Comment:
this is all pretty code-smell-y. i'm not exactly sure what makes sense, but something that's here should probably be in another method.

Reply:
Ideally we'd be using the python keybase api bot.... To reduce additional work for now, I'll make some helper methods for these run_command calls.